forex_python/converter.py

```diff
@@ -96,11 +119,16 @@ def convert(self, base_cur, dest_cur, amount, date_obj=None):
                 converted_amount = rate * amount
                 return converted_amount
             except TypeError:
-                raise DecimalFloatMismatchError("convert requires amount parameter is of type Decimal when force_decimal=True")
+                raise DecimalFloatMismatchError(
+                    "convert requires amount parameter is of type Decimal when force_decimal=True")
+        if response.json()['error']['code'] == "base_currency_access_restricted":
+            raise BaseCurrencyAccessRestricted("Source currency switching is not allowed with this access "
+                                               "key")
         raise RatesNotAvailableError("Currency Rates Source Not Ready")
 
 
-_CURRENCY_FORMATTER = CurrencyRates()
+# Please add your access key here
+_CURRENCY_FORMATTER = CurrencyRates("your_access_key")
```

Comment:
I think the access key should be read from on environtment variable.
This will allow to make it work without alterint the source code.

Probably the same should be done to activate the SSL access.